src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -28,6 +28,11 @@ namespace onnx_mlir {
 
 //====-------------------------- ONNX Builder ---------------------------===//
 
+IntegerAttr OnnxBuilder::getSignedInt64Attr(int64_t n) const {
+  return IntegerAttr::get(b().getIntegerType(64, /*isSigned=*/true),
+      APInt(64, n, /*isSigned=*/true));
+}
+
```

Comment:
This code was manually copied over and over again; got tired of it so I made a local private function. Would you like it to be in the MLIR dialect? If you want a new attribute builder, let's discuss which other functions you may want to have there.

Reply:
yes, MLIR dialect looks ok.

Reply:
@tungld, I decided against putting it into mlir builder, as other wise I need to build that other builder each time I simply want this. If you don't like this, I can put it as a static function (not part of the class) as a simple helper, or remove it all together.

Reply:
OK, let's have another PR to deal with Attribute since this is not the main issue in this patch. Another candidate is something like `Support/TypeUtilities.hpp` that provides utility functions about type.